Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1781,20 +1782,23 @@ static HRESULT _CreatePatternBrush(
 
     size_t bitsPerComponent = 8;
     size_t bytesPerRow = 4 * tileSize.size.width;
-    CGBitmapInfo bitmapInfo = kCGImageAlphaPremultipliedFirst | kCGBitmapByteOrder32Big;
+    CGBitmapInfo bitmapInfo = kCGImageAlphaPremultipliedFirst | kCGBitmapByteOrderDefault;
```

Comment:
How does this work?  We should not be using alphafirst formats since that is not natively supported by d2d.